bundles/org.openhab.binding.serial/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="serial"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Thing Types -->
+	<bridge-type id="serialBridge">
+		<label>Serial Bridge</label>
+		<description>Serial port which can send and receive data</description>
+
+		<channels>
+			<channel id="string" typeId="stringData"/>
+			<channel id="binary" typeId="binaryData"/>
+			<channel id="data" typeId="system.rawbutton"/>
+		</channels>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="true">
+				<context>serial-port</context>
+				<label>Serial Port</label>
+				<description>The serial port to use (e.g. Linux: /dev/ttyUSB0, Windows: COM1)</description>
+			</parameter>
+			<parameter name="baudRate" type="integer">
+				<advanced>true</advanced>
+				<label>Baud Rate</label>
+				<description>Set the baud rate</description>
+				<default>9600</default>
+				<options>
+					<option value="4800">4800</option>
+					<option value="9600">9600</option>
+					<option value="19200">19200</option>
+					<option value="38400">38400</option>
+					<option value="57600">57600</option>
+					<option value="115200">115200</option>
+				</options>
+			</parameter>
+			<parameter name="dataBits" type="integer">
+				<advanced>true</advanced>
+				<label>Data Bits</label>
+				<description>Set the data bits</description>
+				<default>8</default>
+				<options>
+					<option value="5">5</option>
+					<option value="6">6</option>
+					<option value="7">7</option>
+					<option value="8">8</option>
+				</options>
+			</parameter>
+			<parameter name="parity" type="text">
+				<advanced>true</advanced>
+				<label>Parity</label>
+				<description>Set the parity</description>
+				<default>N</default>
+				<options>
+					<option value="N">N(one)</option>
```

Comment:
I think the brackets are not necessary.
```suggestion
					<option value="N">None</option>
```

Reply:
I thought about that, too. But: Usually the settings are given as `8N1` or `7E0`, only one letter. And it's probably more intuitive for less experienced users to have it with the brackets.

Reply:
The use of brackets was copied from the DSMR binding.  Do you want me to keep the brackets or remove them?

Reply:
It's not that important. You can keep them.